tests/Maker/MakeResetPasswordTest.php

```diff
@@ -76,6 +76,49 @@ public function getTestDetails()
             }),
         ];
 
+        yield 'it_generates_with_translator_installed' => [$this->createResetPasswordTest()
+            ->addExtraDependencies('symfony/translation')
+            ->run(function (MakerTestRunner $runner) {
+                $this->makeUser($runner);
+
+                $output = $runner->runMaker([
+                    'App\Entity\User',
+                    'app_home',
+                    'victor@symfonycasts.com',
+                    'SymfonyCasts',
+                ]);
+
+                $this->assertStringContainsString('Success', $output);
+
+                $generatedFiles = [
+                    'src/Controller/ResetPasswordController.php',
+                    'src/Entity/ResetPasswordRequest.php',
+                    'src/Form/ChangePasswordFormType.php',
+                    'src/Form/ResetPasswordRequestFormType.php',
+                    'src/Repository/ResetPasswordRequestRepository.php',
+                    'templates/reset_password/check_email.html.twig',
+                    'templates/reset_password/email.html.twig',
+                    'templates/reset_password/request.html.twig',
+                    'templates/reset_password/reset.html.twig',
+                ];
+
+                foreach ($generatedFiles as $file) {
+                    $this->assertFileExists($runner->getPath($file));
+                }
+
+                $configFileContents = file_get_contents($runner->getPath('config/packages/reset_password.yaml'));
+
+                // Flex recipe adds comments in reset_password.yaml, check file was replaced by maker
+                $this->assertStringNotContainsString('#', $configFileContents);
+
+                $resetPasswordConfig = $runner->readYaml('config/packages/reset_password.yaml');
+
+                $this->assertSame('App\Repository\ResetPasswordRequestRepository', $resetPasswordConfig['symfonycasts_reset_password']['request_password_repository']);
+
+                $this->runResetPasswordTest($runner, 'it_generates_with_normal_setup.php');
```

Comment:
I hope it's OK to reuse this `it_generates_with_normal_setup.php` here because I literally just copy/pasted the `it_generates_with_normal_setup` test completely where just added `addExtraDependencies('symfony/translation')`

Reply:
I think we could remove all the assertions inside of `run()`  except for `$this->assertStringContainsString('Success', $output);` and `$this->runResetPasswordTest($runner, 'it_generates_with_normal_setup.php');`. So, just make it a simpler "smoke test" since the earlier test already covers all the details

Reply:
Makes sense 👍